src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -333,6 +333,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'description': localize('preserveInput', "Controls whether the last typed input to the command palette should be restored when opening it the next time."),
 				'default': false
 			},
+			'workbench.commandPalette.experimental.suggestCommands': {
+				'type': 'boolean',
+				'description': localize('suggestCommands', "Controls whether the command palette should have a list of commonly used commands."),
```

Comment:
@TylerLeonhardt should this be marked as an experimental setting to run an experiment and measure the impact?

Reply:
had a follow up PR doing exactly that.